users/models.py

```diff
@@ -225,82 +225,27 @@ def validate_iso_3166_2_code(value):
 
 
 class LegalAddress(TimestampedModel):
-    """A user's legal address, used for SDN compliance"""
```

Comment:
Can we keep this data model unchanged until we better figure out what we'll be needing around compliance? At a glance that should work fine since you've removed the fields from the serializer and everything can be blank.